src/telliot_ampl_feeds/sources/uspce.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+from telliot.datasource import DataSource
+from telliot.types.datapoint import DataPoint
+from telliot.types.datapoint import datetime_now_utc
+
+
+@dataclass
+class USPCESource(DataSource[float]):
```

Comment:
Man I love this!  Our first plug-in reporter DataSource that takes in a manual input from the user!